news/2018-02-05-project-loved-february-week-1.md

```diff
@@ -0,0 +1,183 @@
+
+Voting is super easy - click the banners to get taken to the voting thread for that map, which contains a poll where you can cast your vote. Any map with enough posts passes muster, and gets in next week.
+
+Let's get stuck in!
+
+### Navigation
+
+<a href="#osu">osu!</a></br><a href="#osutaiko">osu!taiko</a></br><a href="#osumania">osu!mania</a>
+
+### <a name="osu" id="osu"></a>osu!
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!/snow-drive.jpg)](https://osu.ppy.sh/community/forums/topics/699475)
+
+With over 550 favourites, Snow Drive by [jieusieu](https://osu.ppy.sh/u/759439) might be the most popular map in the vote this week, and for good reason. 
+
+The map is ridiculously difficult, clocking at 8.5 stars, and features brutal cross screen jumps as well as a killer spaced stream during the bridge, and has received acclaim from plenty of top players for its playability. Can you survive this lemur of a map?
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!/unravel.jpg)](https://osu.ppy.sh/community/forums/topics/699476)
+
+Returning for its chance at redemption after not passing during the old voting system is [handsome](https://osu.ppy.sh/u/2123087)'s unravel. 
+
+The map sports circle size 2, and features blisteringly high slider velocity with complex 1/4 movement. Despite the difficulty of the concepts used, the map still remains very playable and is definitely worth checking out - if you can handle it!
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!/genesis-at-oasis.jpg)](https://osu.ppy.sh/community/forums/topics/699477)
+
+A full set of a brutal aim map that disguises itself as a stream map. [Krimek](https://osu.ppy.sh/u/2345078)'s Genesis at Oasis is an incredibly high tempo map that builds its intensity towards the end of the map all at a blistering 212 BPM. See if you can handle this behemoth.
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!/foolish-hero.jpg)](https://osu.ppy.sh/community/forums/topics/699478)
+
+[Cassu2](https://osu.ppy.sh/u/2587961)'s Foolish Hero bundles lapix's Hi-Tech style with some interesting high-tech slider patterns themselves. Overall, this proves to be an incredibly enjoyable and popular addition to the Loved vote this week.
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!/mazare-party.jpg)](https://osu.ppy.sh/community/forums/topics/699479)
+
+Mazare Party as a song is a collaboration between 3 popular Japanese vocalists, and likewise the map is a collaboration between 3 popular mappers! 
+
+[ProfessionalBox](https://osu.ppy.sh/u/3250792), [Asahina Momoko](https://osu.ppy.sh/u/3650145), and [Nathan](https://osu.ppy.sh/u/4785223) join forces in a fantastic collab to a fantastic song, with a few lower difficulties by [toybot](https://osu.ppy.sh/u/2848604) and [NeilPerry](https://osu.ppy.sh/u/2848604) to fill out the set.
```

Comment:
I actually missed this one, NeilPerry should be https://osu.ppy.sh/u/841391 (links to toybot profile)